cmp/grid/columns/Column.js

```diff
@@ -380,8 +380,11 @@ export class Column {
                 getReactContainerClasses() {return ['xh-grid-tooltip']}
                 render() {
                     const agParams = this.props,
-                        {value, rowIndex, api} = agParams,
-                        record = api.getDisplayedRowAtIndex(rowIndex).data;
+                        {rowIndex, api, colDef} = agParams,
+                        {headerTooltip} = colDef,
+                        value = isNil(rowIndex) ? headerTooltip : agParams.value;
```

Comment:
This might be more self-documenting here - assuming we are sure the `isNil(rowIndex) == true` only when the mouse is over the header. Just checking that can't happen under any other conditions.
```suggestion
                        overHeader = isNil(rowIndex),
                        value = overHeader ? headerTooltip : agParams.value,
                        record = overHeader ? null : api.getDisplayedRowAtIndex(rowIndex).data
```

Reply:
Correct.  AGGrid says rowIndex == true only when mouse is over header:
"The tooltipComponent detects that it's being rendered by a header because the params object does not contain a rowIndex value." --https://www.ag-grid.com/javascript-grid-tooltip-component/#header-tooltip-with-custom-tooltip